recipe/meta.yaml

```diff
@@ -38,6 +30,9 @@ requirements:
     - python ={{ python_version }}
     - conda ={{ version }}
     - conda-package-handling >=1.6
+    - menuinst >=1.4.18  # [win]
+  run_constrained:
+    - constructor >=3.2.2
```

Comment:
`constructor` supports `--conda-exe` and that can point to _any_ conda-standalone or micromamba binary. This covers only installations in the environment (which are usually for the same host platform). For cross-platform building, it's usually an arbitrary location in disk.

My question is: is `constructor` checking which `conda-standalone` (or compatible) version is bundling? Should it do that? I know this is strictly not part of this PR, but I wondered upon seeing this compatibility constrain here.

Reply:
Very good point and I don't know the answer! The interaction of `conda-standalone` and the `constructor` is not standard at all in the conda ecosystem and I suspect it would be difficult to enforce compatibility.

Documenting it clearly in `constructor` would be a very good first step!

Reply:
We probably need to some introspection in `constructor` to make sure the chosen `conda-exe` supports the needed API before we build the installer.

Reply:
An alternative would be to add an example using menuinst and to run the [`run_example.py`](https://github.com/conda/constructor/blob/master/scripts/run_examples.py) script?

Reply:
Oh, right, we can add it to the tests. I think we can use https://github.com/conda-forge/miniforge_console_shortcut-feedstock for that?

Reply:
Yes!

Reply:
Done: https://github.com/conda/constructor/pull/482